app/src/main/java/com/nervousfish/nervousfish/activities/ContactReceivedHelper.java

```diff
@@ -41,17 +45,22 @@
      * @param contact  The newly received {@link Contact}
      */
     static void newContactReceived(final IDatabase database, final Activity activity, final Contact contact) {
+        if (contact == null) {
```

Comment:
OK, but if you actually read the code you'd have noticed that `contact` is allowed to be `null` 😉